### ISSUE TITLE
Error spam

### ISSUE THREAD

**Issue by Jukabofy** [JMilamber/Brass_Amber_BattleTowers#56]
![image](https://user-images.githubusercontent.com/87483832/148668957-ba822583-564a-4c65-8f88-5ca956526f26.png)
What's going on?
Btw, I love the progress of the mod
 Explain: The first time I defeat a golem tower, it showed some words and then the structure it started to explode, it was in creative mode
Now, in my official survival map, I defeat the miniboss and the chat gets errors, and then it started to explode
That's my issue

**Comment by JMilamber** [comment]
Glad you're enjoying the mod!

3 questions I need answered to be able to fix this:

1. Is this the most recent version of the mod? (1.1.4)
2. What (if any) other mods o you have installed alongside BattleTowers
3. By Official survival map do you mean you are playing on a Dedicated server?

**Comment by Jukabofy** [comment]
Sorry for the incovenience..
Well
1. yes, the last version ![image](https://user-images.githubusercontent.com/87483832/148691599-3622b75f-87db-44e9-adbf-3b107585374a.png)

2.  ![image](https://user-images.githubusercontent.com/87483832/148691644-1d26f988-bfd2-436e-aa9d-9a596f85aa19.png)
This is my mod list

3. I mean, singleplayer XD
4. REMEMBER, in creative mode seems to work well, and in survival it showed me that messages, I didn't change anything in my mod list or config, just updating mods, like Relics and Ender Cataclysm

**Comment by Jukabofy** [comment]
![image](https://user-images.githubusercontent.com/87483832/148691950-b0663750-b65a-41d7-b8aa-4a0da2c8ab74.png)

I will wait until the fix, I like this mod so much 😄

**Comment by JMilamber** [comment]
Sorry if I came off rude in my first reply! 

Thanks for the info! I have been working on another project this weekend, I'll hopefully have a fix for this by Tuesday/Wednesday! 

Thanks for your patience! <3

**Comment by JMilamber** [comment]
Apologies, I did not get around to fixing this today*, 100% looking at it tomorrow*, top of the list lol xD 

Today = Wednesday 
Tomorrow = Thursday

**Comment by JMilamber** [comment]
I honestly have no idea why this is happening :/
couldn't reproduce it with BT alone, so I believe it is a mod conflict somewhere. 
I looked through all of the mods on your list that I could imagine being a problem (mostly the API mods cuz they sometimes mess with chat/command stuff) but they all worked fine in tandem, or at least should be based on their code. 

At this point I have to ask for a favor since I don't have the time to do this personally: 
    Basically at this point, the only thing that makes sense is that it is a mod conflict - so, I'm gonna have to ask pretty please if you could try removing the mods in your pack 1 by 1 until this error no longer occurs. 
    
Then tell me the mod that's causing it and I can get in contact with them and see if it is fixable. 
    
Apologies, I know that is gonna be a lot of work since you have so many mods installed xD and if you don't feel up to it, all good I don't blame you lol. 

The mods I explicitly checked by running in Minecraft- 
Abnormals_Core
Environmental
Majrusz-library
Cuniform
Explorers compass (I have used alongside BT before)

Mods I checked source code of but did not run in Minecraft itself alongside BT -
ADFinders
Mcwifipnp
Project vibrant journeys
Mirabilis
Ezmodlib
Cloth-config
Dynamic Surroundings
fabricBiomeApirReforged
